### DISCUSSION TITLE
My proof submission screen is different from what is shown in Q&A, I can't select the type of proof, and I've been rejected multiple times for the wrong type. How can I fix this please?

### DISCUSSION THREAD

**Discussion by PSGLGDFaithbian7** [community/community#77323] [GitHub Education]
### Hi everyone,

<img width="694" alt="屏幕截图 2023-11-29 234820" src="https://github.com/community/community/assets/129975898/67632767-483a-44cf-80f3-ab2702d70bec">
My proof submission screen is different from what is shown in Q&A, I can't select the type of proof, and I've been rejected multiple times for the wrong type. How can I fix this please?

**Comment by YourSonnet18** [comment]
same problem

**Reply by YourSonnet18** [reply]
It's been a while and the problem still exists.

**Comment by shangji666** [comment]
same problem
